File: solutions/0283-move-zeroes/solution.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        n=[i for i in nums if i!=0]
        n=n+[0]*(len(nums)-len(n))
        for i in range(0,len(nums)):
            nums[i]=n[i]

File: solutions/0283-move-zeroes/solution.java
class Solution {
    public void moveZeroes(int[] nums) {
        int write=0,n=nums.length;
        for(int i=0;i<n;i++)
            if(nums[i]!=0)
                nums[write++]=nums[i];
        for(;write<n;write++)
            nums[write]=0;
    }
}
